community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -537,14 +538,45 @@ def _allocate_nodes_to_placements(nodes: List[str], excl_job_id:Optional[int], l
 
     return placements
 
+def calculate_hosts_per_topo(gpu_topology: str, machine_type: NSDict) -> int:
+    # Calculate total number of hosts per topology (Assumes format: '1x72')
+    try:
+        top_split = [int(x) for x in str(gpu_topology).split("x")]
+    except Exception as e:
+        log.error("Incorrectly formatted accelerator topology")
```

Comment:
All errors / exceptions should contain the accelerator_topology value to be useful.